conda_forge_tick/update_upstream_versions.py

```diff
@@ -9,11 +9,63 @@
 from conda.models.version import VersionOrder
 from pkg_resources import parse_version
 
-from .utils import parse_meta_yaml, setup_logger
+from utils import parse_meta_yaml, setup_logger
+import os
 
 logger = logging.getLogger("conda_forge_tick.update_upstream_versions")
 
 
+class LibrariesIO:
+
+    def __init__(self):
+        self.libraries_io_api_key = os.getenv("LIBRARIES_IO_API_KEY")
+        self._data = None
+
+    def get_libraries_io_subscriptions(self):
+        if self.libraries_io_api_key == None:
+            return
+        l = []
+        for i in range(1, 2):
```

Comment:
Won't `i` always be 1 in this loop? It looks like you actually want something like
```
i = 0
while True:
    do stuff
    i += 1
```